src/main/java/ca/wolfram/beta/symath/operations/PowerNode.java

```diff
@@ -8,15 +8,16 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class PowerNode extends OperationNode{
+public class PowerNode extends OperationNode {
 
     private PowerNode(List<MathNode> children) {
         super(children);
     }
 
     /**
      * Index 0 is the base and Index 1 is the power.
-     * @throws RuntimeException if number of nodes is incorrect.
+     *
+     * @throws IllegalArgumentException if number of nodes is incorrect.
```

Comment:
I feel like the throwing part shouldn't even be here. The checker is in OperationNode, not just PowerNode